tests/test_zones.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2018 Comcast Cable Communications Management, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add module docstring."""
+
+import json
+from datetime import datetime
+
+import responses
+
+from sampledata import acl_rule, forward_zone, ip4_zone, ip6_zone, sample_zone_change
+from vinyldns.serdes import to_json_string, from_json_string
+from vinyldns.zone import Zone, ZoneChange, ListZonesResponse, ListZoneChangesResponse
+
+
+def check_zones_are_same(a, b):
+    assert a.id == b.id
+    assert a.name == b.name
+    assert a.email == b.email
+    assert a.admin_group_id == b.admin_group_id
+    assert a.status == b.status
+    assert a.updated == b.updated
+    assert a.created == b.created
+    assert a.connection.primary_server == b.connection.primary_server
```

Comment:
Might be better to have a `check_zone_connections_are_the_same` to do an assert based off of `None` or existing fields, which we could call for both primary and transfer connections.

Reply:
True, this is definitely incomplete, I will augment